src/taipy/core/taipy.py

```diff
@@ -497,3 +498,51 @@ def export_scenario(
         _ScenarioManagerFactory._build_manager()._export(scenario_id, folder_path)
     for job_id in entity_ids.job_ids:
         _JobManagerFactory._build_manager()._export(job_id, folder_path)
+
+
+def get_parents(
+    entity: Union[TaskId, DataNodeId, PipelineId, Task, DataNode, Pipeline], parent_dict=None
+) -> Dict[str, Set[_Entity]]:
+    """Get parents of the entity from the entity or its identifier.
```

Comment:
```suggestion
    """Get parents of an entity from the entity or its identifier.
```

Reply:
@jrobinAV 
I'll change it to `Get the parents of an entity from itself or its identifier.`, what do you think?

Reply:
Yes, it s better.